bundles/net.tourbook/src/net/tourbook/data/TourData.java

```diff
@@ -9587,17 +9587,19 @@ public int[] getRecordedTimeSerie() {
                   final int timeDiff = nextRelativeTime - relativeTime;
 
                   // set pause in the next slice because the pause flag is set before the pause
-                  nextSlicePausedTime = isAutoPause
+                  nextSlicePausedTime = // isAutoPause
 
 //                      // auto pauses are ignored
 //                      ? 0
 
                         // auto pauses are not ignored
                         // https://github.com/mytourbook/mytourbook/issues/502#issuecomment-1498240431
-                        ? timeDiff
+                        //? timeDiff
 
                         // pause is triggered by a user
-                        : timeDiff;
+                        //: timeDiff;
+
+                        timeDiff;
```

Comment:
Indeed, my commit title doesn't help.

I addressed [this issue](https://sonarcloud.io/project/issues?issues=AYd5mIdF5rDj81LnQu7_&open=AYd5mIdF5rDj81LnQu7_&id=wolfgang-ch_mytourbook) where the same result happens in both cases, which creates a little performance loss. 
I fixed it but didn't remove the code nor the comments

Reply:
I agree with SonarCloud, the code was very confusing because when reading it, it felt like it was a bug to have the same result for each branch.

![image](https://github.com/mytourbook/mytourbook/assets/25235665/6963a47a-06af-42af-949f-b0730471e80b)

Reply:
> ... but didn't remove the code nor the comments

👍 

for this code it's good to see the history